.github/scripts/end2end/configs/zenkoversion.yaml

```diff
@@ -58,8 +58,8 @@ spec:
       image: '${CLOUDSERVER_IMAGE}' 
       tag: '${CLOUDSERVER_TAG}'
     backbeat:
-      image: '${BACKBEAT_IMAGE}' 
-      tag: '${BACKBEAT_TAG}'
+      image: 'registry.scality.com/playground/kgardahaut/backbeat' 
+      tag: '8.6.25'
```

Comment:
Leaving a note to not forget reverting this and bump backbeat instead when it's released